addons/account/models/res_config_settings.py

```diff
@@ -88,6 +104,19 @@ def _compute_has_chart_of_accounts(self):
         self.chart_template_id = self.company_id.chart_template_id or False
         self.has_accounting_entries = self.env['wizard.multi.charts.accounts'].existing_accounting(self.company_id)
 
+    @api.onchange('sale_show_tax')
+    def _onchange_sale_tax(self):
```

Comment:
Here you overwrite the method. (Without even calling to super). 

Could be removed.

Reply:
Same here, my bad.